lib/active_support/cache/memcached_store.rb

```diff
@@ -71,16 +56,18 @@ def read_multi(*names)
         values = {}
 
         instrument(:read_multi, names, options) do
-          if raw_values = @data.get_multi(keys_to_names.keys, raw: true)
+          if raw_values = @data.get(keys_to_names.keys, false)
```

Comment:
Oh, and yes... `Memcached#get` will just "multi" or "single" based on the `get`'s `keys` arg type:

```
  def get(keys, decode=true)
    if keys.is_a? Array
      multi_get(keys, decode).first
    else
      single_get(keys, decode).first
    end
```